### PR TITLE
Doc: Update GSG for v0.4 version and launch and acrn.conf sample script

### PR DESCRIPTION
**Pull request by NanlinXie**
Doc: Update GSG for v0.4 version and launch and acrn.conf sample script

Signed-off-by: Xie, nanlin <nanlin.xie@intel.com>

### CODE REVIEW

File: doc/getting-started/apl-nuc.rst [hunk 1]
```diff
@@ -118,9 +118,9 @@ partition. Follow these steps:
 
       $ sudo ls -1 /boot/EFI/org.clearlinux
       bootloaderx64.efi
-      kernel-org.clearlinux.native.4.19.1-654
-      kernel-org.clearlinux.iot-lts2018-sos.4.19.0-19
-      kernel-org.clearlinux.iot-lts2018.4.19.0-19
+      kernel-org.clearlinux.native.4.19.8-670
+      kernel-org.clearlinux.iot-lts2018-sos.4.19.5-29
+      kernel-org.clearlinux.iot-lts2018.4.19.5-29
       kernel-org.clearlinux.pk414-sos.4.14.74-115
```
Comment (gvancuts) [inline doc/getting-started/apl-nuc.rst]: Do we still have that `pk414` kernel installed (via the `service-os` bundle)? If not, can you remove this entry?

**Comment by HuiHuangShi** [comment]
2071 <br>The PR does not touch any hypervisor source file. Skip coding guideline checks.<br>

**Comment by gvancuts** [comment]
@NanlinXie , I see you're pushing new versions (the last one 12 mins ago), but I don't see any difference compared to what you submitted initially... am I missing something or could it be that you forgot to `git add` the files updated before the `git commit --amend`?

**Comment by dbkinder** [comment]
when this get's approved, do we need to apply this PR to both master and the 0.4 tag?

**Comment by NanlinXie** [comment]
@gvancuts @dbkinder This PR is just for update E2E version number that including ACRN v0.4 tag in GSG, guiding user how to use ACRN v0.4.  Thus this is not needed to included in v0.4 tag, but hopefully be merged and published ASAP. @gvancuts, for your advice on other change in GSG, let's put it into v0.5 scope.

**Comment by dbkinder** [comment]
Given the urgency, I'll merge this and as @NanlinXie suggests, further changes can be made with another PR later.

**Pull request merged**